config/boards/matebook-e-go.conf

```diff
@@ -0,0 +1,185 @@
+# Qualcomm Snapdragon 8cx Gen 3 Adreno 690 Qualcomm WCN685x Wi-Fi 6E Qualcomm Snapdragon X55 5G Bluetooth 5.1
+declare -g BOARD_NAME="HUAWEI MateBook E Go"
+declare -g BOARD_MAINTAINER="chenxuecong"
+declare -g BOARDFAMILY="uefi-arm64"
+declare -g KERNEL_TARGET="sc8280xp"
+
+declare -g BOOT_LOGO=desktop
+
+declare -g GRUB_CMDLINE_LINUX_DEFAULT="efi=noruntime clk_ignore_unused pd_ignore_unused arm64.nopauth iommu.passthrough=0 iommu.strict=0 pcie_aspm.policy=powersupersave modprobe.blacklist=msm"
+declare -g BOOT_FDT_FILE="qcom/sc8280xp-lenovo-thinkpad-x13s.dtb"
```

Comment:
It is using devicetree of thinkpad x13s. So how similar are these two?

Reply:
except device tree